backend-shared/exportfuncs.cpp

```diff
@@ -30,6 +32,28 @@ bool ExportCallback::canceled() const
 }
 
 #if !defined(SUBSURFACE_MOBILE)
+
+#include "desktop-widgets/mainwindow.h" // Currently needed for profile printing. TODO: remove.
+
+static void exportProfile(const struct dive *dive, const QString filename)
+{
+	ProfileWidget2 *profile = MainWindow::instance()->graphics;
```

Comment:
the last two commits created separate profiles for printing purposes. Why is this one reusing the `MainWindow::instance()->graphics` ?
Or am I just not reading far enough ahead and this is changed in a later commit?